src/kili/adapters/kili_api_gateway/label/operations.py

```diff
@@ -1,6 +1,39 @@
 """GraphQL label operations."""
 
 
+from typing import Dict, List
+
+from kili.adapters.kili_api_gateway.helpers.queries import (
+    fragment_builder,
+)
+from kili.core.graphql.graphql_client import GraphQLClient
+from kili.domain.label import LabelId
+from kili.domain.types import ListOrTuple
+
+
+def list_annotations(
```

Comment:
it is strange to put that function here, as in general only mutations and queries are listed. If this is because it is used in several places of the gateway, this deserves to be in a `label/common.py` or `label/utils.py` file.

Reply:
this can also be inside a private method of the `kili_api_gateway`

Reply:
I put it in common.py